addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -124,14 +124,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
+        buttonExecutor = Executors.newSingleThreadScheduledExecutor();
         cancelRetry();
         connect();
     }
 
     @Override
     public void dispose() {
-        listeners.clear();
         buttonExecutor.shutdownNow();
```

Comment:
But where do you shutdown the executor threads, if not in `dispose()`? The binding could handle configuration updates on its own and therefore avoid additional dispose/initialize sequences.